plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -729,8 +729,12 @@ def update_needed(self, other):
         err_msg = _('Fail to update the %(which)s erratum %(id)s.')
         existing_err_msg = err_msg % {'which': 'existing', 'id': self.errata_id}
         other_err_msg = err_msg % {'which': 'uploaded', 'id': self.errata_id}
-        existing_updated_dt = util.errata_format_to_datetime(self.updated, msg=existing_err_msg)
-        new_updated_dt = util.errata_format_to_datetime(other.updated, msg=other_err_msg)
+        try:
+            existing_updated_dt = util.errata_format_to_datetime(self.updated, msg=existing_err_msg)
+            new_updated_dt = util.errata_format_to_datetime(other.updated, msg=other_err_msg)
+        except ValueError as e:
+            _LOGGER.warn(str(e))
```

Comment:
Rather than logging each one individually here, @mhrivnak and I propose doing the following instead:

1. Let the `ValueError` Exception get raised normally through the call the `update_needed()`.
2. Have the caller to `update_needed()` treat a `ValueError` as the same as a return value of `False`
3. Count the number of total exceptions (for example it is 3624 in [this comment](https://github.com/pulp/pulp_rpm/pull/925#issuecomment-230497085))
4. Log exactly 1 warning statement showing that XXXX errata (e.g. 3624) could not have the update field parsed, and if the those errata have been modified on the remote repository that those changes have not been pulled in locally.

Maybe also add to the log statement (4) that users can work the the owners of the remote repository and ask them to publish correct data in the updated field.